cg/services/orders/order_service/order_service.py

```diff
@@ -40,3 +40,9 @@ def create_order(self, order_data: OrderIn) -> OrderResponse:
 
     def set_delivery(self, order_id: int, delivered: bool) -> None:
         self.store.update_order_delivery(order_id=order_id, delivered=delivered)
+
+    def update_delivered(self, order_id: int, delivered_analyses: int) -> None:
```

Comment:
I think we should add support for unmarking as delivered, say if someone accidentally marks an analysis as delivered and immediately unmarks it?

Reply:
That is a separate endpoint implemented here https://github.com/Clinical-Genomics/cg/pull/3095

Reply:
@seallard no? Is that not for manual markings of the order's delivery status? I would want this to happen when an analysis is unmarked as delivered.

Reply:
Ah, nice! Good point 👍

Reply:
Added in https://github.com/Clinical-Genomics/cg/pull/3097/commits/f7bc777afbec6e449897c3e7c43ffd2164028004